package.json

```diff
@@ -1,7 +1,13 @@
 {
   "name": "gitcoin",
   "version": "1.0.0",
-  "description": "pushing open source forward",
+  "description": "Gitcoin is the easiest way to monetize or incentivize work in Open Source Software",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": "eslint --quiet --config .eslintrc ./app/assets/v2/js/**/*.js",
+    "stylelint": "stylelint ./app/assets/**/**/*.*css",
+    "build": "webpack --config webpack.config.js --progress --colors"
```

Comment:
The bug title said it's about convert to es6, why need to introduce the webpack build here?

I also suggest add "lint": "npm run eslint && npm run stylelint" for all lints.
Can add `npm run lint` command into `travis.yml` to force all future patch pass the linting requirements, but that could be a separate PR

Reply:
The other part of es6 was to make it modular and to have imports.  Is there a better way?

Thanks for noticing that, we can definitely add that and I suppose that could be part of the Travis PR.

Yes the addition to travis is going to be another PR.